### PR TITLE
Add alerts to cloudformation/AWS

### LINKED ISSUE #29: Create alarm automatically for 5xx queries, redis memory
I'm self-hosting a AWS stack, and it went down for a day without me noticing until hitting the node.

It would be nice if I could have received an email about these issues.

Relevant documentation: https://docs.aws.amazon.com/awscloudtrail/latest/userguide/use-cloudformation-template-to-create-cloudwatch-alarms.html

### PR DESCRIPTION
**Pull request by macobo**
Add alerts for elasticcache memory full and RDS disk full. 

This should notify our users when things are going sideways
    
Note users will need to manually confirm their subscriptions for these alerts - couldn't find an automatic way to do this.
    
Closes #29

Some emails:
Screenshot from 2020-11-05 09-51-59

Screenshot from 2020-11-05 10-04-04

### CODE REVIEW

File: aws/cloudformation/ecs/posthog.yaml [hunk 1]
```diff
@@ -864,9 +873,57 @@ Resources:
...
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmDescription: "Redis memory over the last 10 minutes is too low, your instance might not be ingesting events"
+      AlarmActions:
+      - Ref: PosthogAlarmTopic
+      MetricName: FreeableMemory
+      Namespace: AWS/RDS
+      Statistic: Average
+      Period: 600 # 10 minutes
+      Threshold: 640000000 # Less than 64MB
```
Comment (yakkomajuri) [inline aws/cloudformation/ecs/posthog.yaml:917]: We have a lot of hobby users, deploying PH for small projects or just to try it out. Would this not trigger for them?

Also, would this not trigger if I set up PostHog and don't ingest events in the first 10min?

Could be my ignorance here as to how low 64mb for redis actually is.

**Review by yakkomajuri — COMMENTED** [review]
Minor comment but otherwise looks good to me.

File: aws/cloudformation/ecs/posthog.yaml (hunk 1 above)
Comment (macobo) [reply aws/cloudformation/ecs/posthog.yaml:917]: So redis runs in memory/in ram. The smallest redis instance available in AWS has 555mb of ram. https://aws.amazon.com/elasticache/pricing/ 

I think this is fair.

**Comment by macobo** [comment]
@yakkomajuri thanks for the review! Merging

**Pull request merged**